src/form/time/select/TimeSelect.tsx

```diff
@@ -0,0 +1,102 @@
+import "./_time-select.scss";
+
+import React, {useState} from "react";
+import classNames from "classnames";
+
+import {formatDateWithOptions} from "../../../core/utils/time/timeUtils";
+import {DATE_FORMAT} from "../../../core/utils/time/timeConstants";
+import {DropdownOption} from "../../../dropdown/list/item/DropdownListItem";
+import TimeInput from "../input/TimeInput";
+import TimeDropdown from "../dropdown/TimeDropdown";
+
+export type TimeSelectDropdownOption = DropdownOption<
+  string,
+  {date: Date; differenceInMinutes: number}
+>;
+
+export interface TimeSelectProps {
+  onChange: (timeString: string) => void;
```

Comment:
I think we would want to pass a "selectedTime" prop instead of `startTime` and `startDate`. This way the props make more sense and it is obvious that TimeSelect is actually a controlled component.

Reply:
We removed `TimeSelect` and `TimeDropdown` component as we discussed.